src/components/AccountDetailsModal/index.tsx

```diff
@@ -161,11 +159,11 @@ export const AccountDetailsModal: React.FC<IAccountDetailsModal> = ({
     try {
       const storagePrimaryAssets = localStorage.getItem('primaryAsset');
       if (storagePrimaryAssets) {
```

Comment:
use guard clause to improve the conditional flow
https://deviq.com/design-patterns/guard-clause